spring-cloud-skipper-server-core/src/test/java/org/springframework/cloud/skipper/server/service/ArgumentSanitizerTests.java

```diff
@@ -36,7 +39,7 @@ public void testNoChange() throws Exception {
 				TestResourceUtils.qualifiedResource(getClass(), "nopassword.yaml").getInputStream(),
 				Charset.defaultCharset());
 		String result = ArgumentSanitizer.sanitizeYml(initialYaml);
-		assertThat(result.replace("\r\n", "\n")).isEqualTo(initialYaml.replace("\r\n", "\n"));
+		assertThat(LineUtil.removeCr(result)).isEqualTo(LineUtil.removeCr(initialYaml));
```

Comment:
This is an improvement over the previous version but I would prefer to abstract it out further so that I don't have to know to use LineUtil on each operand. Something like I pointed out in the previous comments.

Reply:
Updating